FE/src/components/Account.jsx

```diff
@@ -1,8 +1,412 @@
+            ],
+            procedure: [
+              "Add the mayonnaise, sour cream, buttermilk, vinegar, and worcestershire sauce to a bowl and mix until well combined.",
+              "Crumble in the Blue Cheese and add your choice of fresh herbs and the minced garlic.",
+              "Season with the Salt and Black pepper to taste.",
+              "For best results, chill before serving.",
+            ],
+            imgUrl:
+              "https://img.freepik.com/free-photo/top-view-yogurt-meal-brown-wooden-desk-food-yogurt-meat_140725-28373.jpg?w=740&t=st=1707958620~exp=1707959220~hmac=ccb6ad132464b618af23d47c6bbd510185020066158580387806fbb896d3c72c",
+            notes: [
+              "Pick a really good blue cheese for this and you'd be amazed how excellent this can be.",
+            ],
+            tags: ["condiment"],
+          }
+    ]
+
+    const dummyReviews = [
+        {
+            id: 1,
+            userid: 1,
+            recipeid: 1,
+            content: 'This is cool',
+            rating: 4,
+            user: {
+                id: 1,
+                email: 'adminTest@gmail.com',
+                username: 'adminTest',
+                name: 'Admin Test',
+                imgurl: 'https://www.icb.org.za/wp-content/uploads/2023/08/2023-Blog-header-images-10-1030x579.png',
+                admin: true,
+                reviewcount: 0
+            },
+            recipe: {
+                id: 1,
+                userid: 2,
+                title: 'New Title',
+                ingredients: [Array],
+                procedure: [Array],
+                notes: [Array],
+                imgurl: 'https://www.foodiesfeed.com/wp-content/uploads/2023/03/close-up-of-butter-chicken-indian-dish.jpg'
+            }
+        }
+    ]
+    const dummyComments = [
+        {
+            id: 1,
+            userid: 1,
+            reviewid: 5,
+            content: 'thanks bruh',
+            user: {
+              id: 1,
+              email: 'adminTest@gmail.com',
+              username: 'adminTest',
+              name: 'Admin Test',
+              imgurl: 'https://www.icb.org.za/wp-content/uploads/2023/08/2023-Blog-header-images-10-1030x579.png',
+              admin: true,
+              reviewcount: 0
+            },
+            review: {
+              id: 5,
+              userid: 2,
+              recipeid: 7,
+              content: 'This is cool 2',
+              rating: 4
+            },
+            recipe: {
+              id: 7,
+              userid: 6,
+              title: 'Tasty Chicken',
+              ingredients: [Array],
+              procedure: [Array],
+              notes: [Array],
+              imgurl: 'https://www.foodiesfeed.com/wp-content/uploads/2023/03/close-up-of-butter-chicken-indian-dish.jpg'
+            }
+        }
+    ]
+        
+    useEffect(()=>{
+        async function userCheck () {
+            try {
+                // const response = await fetch(`${API}users/me`, {
```

Comment:
Noting here that we should probably store the API url in a .env file so we can change it from localhost to the hosted api later. This is nothing you need to do now, just a thought for later.